collector/lib/SensorClientFormatter.cpp

```diff
@@ -0,0 +1,352 @@
+#include "SensorClientFormatter.h"
+
+#include <uuid/uuid.h>
+
+#include <google/protobuf/util/time_util.h>
+
+#include "internalapi/sensor/collector_iservice.pb.h"
+#include "internalapi/sensor/signal_iservice.pb.h"
+
+#include "CollectorStats.h"
+#include "EventMap.h"
+#include "Logging.h"
+#include "Utility.h"
+#include "system-inspector/EventExtractor.h"
+
+namespace collector {
+
+using SignalStreamMessage = sensor::SignalStreamMessage;
+using Signal = SensorClientFormatter::Signal;
+using ProcessSignal = SensorClientFormatter::ProcessSignal;
+using LineageInfo = SensorClientFormatter::LineageInfo;
+
+using Timestamp = google::protobuf::Timestamp;
+using TimeUtil = google::protobuf::util::TimeUtil;
+
+namespace {
```

Comment:
Not saying this is wrong, but how did you decide to use an anonymous namespace rather than declaring these as private?